web_generator/lib/src/config.dart

```diff
@@ -174,3 +185,14 @@ class YamlConfig implements Config {
             []);
   }
 }
+
+/// Creates the `exclude` function for [FSGlobSyncOptions]'s `exclude` option
+/// to exclude all files not ending with the given extension
+///
+/// This helps support passing dir globs.
+bool Function(FSDirent entry) excludeFileEntryFunc(String extension) {
```

Comment:
nit: `allowedExtension` instead of `extension` so it's obvious that it's being included not excluded